impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -33,6 +39,62 @@ namespace impeller {
 //              https://github.com/flutter/flutter/issues/114563
 constexpr auto kPadding = 2;
 
+namespace {
+
+class HostBufferAllocator : public SkBitmap::Allocator {
+ public:
+  explicit HostBufferAllocator(HostBuffer& host_buffer)
+      : host_buffer_(host_buffer) {}
+
+  BufferView TakeBufferView() {
+    buffer_view_.buffer->Flush();
+    return std::move(buffer_view_);
+  }
+
+  bool allocPixelRef(SkBitmap* bitmap) {
```

Comment:
Is this an override? Maybe add the override and decorate it with the comment for an override.

Reply:
Done